ecs-logs-collector.sh

```diff
@@ -552,6 +601,28 @@ get_ecs_agent_info() {
     fi
   fi
 
+  if [ -e /etc/ecs/ecs.config ]; then
+    source_file="/etc/ecs/ecs.config"
+
+    line_is_safe=0
+    while IFS= read -r line; do
+      if [[ "$line" == *"="* ]]; then
+        var_name="${line%%=*}"
+        if [[ -v ecs_config_allowlist["$var_name"] ]]; then
+          line_is_safe=1
```

Comment:
Can we just do `echo "$line" >> "$info_system"/ecs-agent/ecs.config` here, and then `line_is_safe` and the conditional on lines 619-621 are no longer necessary?

Reply:
Thanks for the explanation!